ansible/templates/docker-compose.yml

```diff
@@ -25,12 +25,13 @@ services:
       - ./volumes/postgres:/var/lib/postgresql/data
     restart: always
 
-  pictshare:
-    image: hascheksolutions/pictshare:latest
+  pictrs:
+    image: asonix/pictrs:amd64-v0.1.0-r9
```

Comment:
You need to update to the latest version before merging, and ideally make it work on all architectures.